packages/web-forms/src/demo/FormPreview.vue

```diff
@@ -50,17 +55,27 @@ xformResource
 		alert('Failed to load the Form XML');
 	});
 
-const handleSubmit = () => {
+const handleSubmit = (payload: MonolithicSubmissionResult) => {
+	// eslint-disable-next-line no-console
+	console.log('submission payload:', payload);
+
 	alert(`Submit button was pressed`);
 };
+
+const handleSubmitChunked = (payload: ChunkedSubmissionResult) => {
+	// eslint-disable-next-line no-console
+	console.log('CHUNKED submission payload:', payload);
+};
 </script>
 <template>
 	<template v-if="formPreviewState">
 		<OdkWebForm
 			:form-xml="formPreviewState.formXML"
 			:fetch-form-attachment="formPreviewState.fetchFormAttachment"
 			:missing-resource-behavior="formPreviewState.missingResourceBehavior"
+			:submission-max-size="Infinity"
```

Comment:
Question:

Is there any limit in some other place, like Central dropping the connection or rate limit, after a while of long or many chucks? Asking in case this is a configuration setting.

Reply:
> but I'd also be happy to put that off until we have support for submission attachments, where we'll actually get chunked submissions working 

Sounds good! Thank you